mappings/net/minecraft/server/world/ServerChunkManagerEntry.mapping

```diff
@@ -0,0 +1,23 @@
+CLASS tz net/minecraft/server/world/ServerChunkManagerEntry
+	CLASS tz$a Unloaded
+		FIELD a INSTANCE Ltz$a;
+	CLASS tz$b
+	FIELD a UNLOADED_CHUNK Lcom/mojang/datafixers/util/Either;
+	FIELD b UNLOADED_CHUNK_FUTURE Ljava/util/concurrent/CompletableFuture;
+	FIELD c UNLOADED_WORLD_CHUNK Lcom/mojang/datafixers/util/Either;
+	FIELD d UNLOADED_WORLD_CHUNK_FUTURE Ljava/util/concurrent/CompletableFuture;
+	FIELD e chunkStatuses Ljava/util/List;
```

Comment:
Should this be `UPPER_SNAKE_CASE`? It seems Yarn is following this pattern for `private static final` fields in other places, but I have not checked that much.

Reply:
Yes, we do, for all static final fields